src/test/java/de/mediathekview/mserver/crawler/basic/M3U8DtoTest.java

```diff
@@ -0,0 +1,96 @@
+package de.mediathekview.mserver.crawler.basic;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+import org.junit.Test;
+
+public class M3U8DtoTest {
+
+  @Test
+  public void equalTestSameObject() {
+    M3U8Dto target = new M3U8Dto("test");
+    
+    boolean actual = target == target;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Correct one of the identical sub-expressions on both sides of operator "==" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1764)